From: Ibrahim ARI <ibrahim.ari@pardus.org.tr>
Date: Thu, 3 Jan 2019 16:27:48 +0300
Subject: added mbuild parameters for dnp3

---
 extensions/Mbuild | 1 +
 1 file changed, 1 insertion(+)

diff --git a/extensions/Mbuild b/extensions/Mbuild
index 7069389..90510d5 100644
--- a/extensions/Mbuild
+++ b/extensions/Mbuild
@@ -23,4 +23,5 @@ obj-${build_psd}         += libxt_psd.so
 obj-${build_quota2}      += libxt_quota2.so
 obj-${build_gradm}       += libxt_gradm.so
 obj-${build_modbus}      += libxt_modbus.so
+obj-${build_dnp3}        += libxt_dnp3.so
 
